### PR TITLE
New version: GitTools.GitVersion version 5.12.0

### PR DESCRIPTION
**Pull request by arturcic**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/121108)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Exception during executable launch operation System.AggregateException: One or more errors occurred. (The specified executable is not a valid application for this OS platform.)  ---
> System.ComponentModel.Win32Exception (216): The specified executable is not a valid application for this OS platform. 
--- End of stack trace from previous location where exception was thrown ---    --- End of inner exception stack trace ---    at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)

File: manifests/g/GitTools/GitVersion/5.12.0/GitTools.GitVersion.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- gitversion
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Git.Git
+ReleaseDate: 2023-01-20
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/GitTools/GitVersion/releases/download/5.12.0/gitversion-win-x64-5.12.0.zip
+  InstallerSha256: AF2EC460E3C112BB09B793CC0AE691748DE2EE5742D1CBC4B996EFC78C7B4597
+- Architecture: x64
```
Comment (stephengillie) [inline manifests/g/GitTools/GitVersion/5.12.0/GitTools.GitVersion.installer.yaml:22]:
```suggestion
- Architecture: x64
  InstallerUrl: https://github.com/GitTools/GitVersion/releases/download/5.12.0/gitversion-win-x64-5.12.0.zip
  InstallerSha256: AF2EC460E3C112BB09B793CC0AE691748DE2EE5742D1CBC4B996EFC78C7B4597
- Architecture: x86
```

**Comment by arturcic** [comment]
@stephengillie it seems the [Komac](https://github.com/russellbanks/Komac) tool is setting the wrong artifact for x86, and x64 in the manifest for some reason, had to manually fix it.

Can you tell me what the Validation-Executable-Error label means? and what I need to do to fix this?

Thanks

**Comment by stephengillie** [comment]
Hi @arturcic,

The `Validation-Executable-Error` here was because our validation process is designed to work with Windows Forms, and CLI applications confuse it.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**